watchdog/src/reporters/metrics.go

```diff
@@ -0,0 +1,47 @@
+package reporters
+
+import (
+	"bytes"
+	"encoding/base64"
+	"log"
+	"net/http"
+
+	"github.com/lumigo-io/lumigo-kubernetes-operator/watchdog/config"
+)
+
+type MetricsReporter struct {
+	endpoint string
+	config   *config.Config
+}
+
+func NewMetricsReporter(config *config.Config) *MetricsReporter {
+	return &MetricsReporter{
+		endpoint: config.LUMIGO_ENDPOINT + "/api/v1/",
+		config:   config,
+	}
+}
+
+func (r *MetricsReporter) Report(metrics string) {
+
+	url := r.endpoint + "metrics"
+	token := r.config.LUMITO_TOKEN
+
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer([]byte(metrics)))
+	if err != nil {
+		log.Println("Error creating request:", err)
+		return
+	}
+	encodedToken := base64.StdEncoding.EncodeToString([]byte(token))
+	req.Header.Set("Authorization", "Bearer "+encodedToken)
```

Comment:
should we use `LumigoToken <token>` instead of Bearer, when sending to our backend?

Reply:
in which format is the data being exported? we need it to be otlp v1/metrics preferably